bokehjs/src/lib/models/ranges/data_range1d.ts

```diff
@@ -168,6 +168,13 @@ export class DataRange1d extends DataRange {
     const range_padding = this.range_padding // XXX: ? 0
 
     let start, end: number
+
+    if (this._initial_start != null)
+      min = Math.min(min, this._initial_start)
+
+    if (this._initial_end != null)
+      max = Math.max(max, this._initial_end)
+
```

Comment:
@Tikonderoga thanks for your patience while we got 2.0 out the door. What is the rationale for using `Math.min` and `Math.max` here? Offhand I think I would expect if `this._initial_start` is defined then it should just be used as the value unconditionally.

Reply:
Hello, @bryevdv 

I've re-tested both versions (mine initial and yours) and yours works makes more sense and produces more consistent representation. I will change code.

I think I should re-base my branch on 2.0, right?

Reply:
> I think I should re-base my branch on 2.0, right?

Preferably yes, though there weren't many changes since then.

Reply:
Thanks, @mattpap 
I'll create new pull request

Reply:
@mattpap 

I've forked 2.0 and I can't make it work.
I've created new fork, followed setup guide and build bokehjs.

But as soon as I'm trying to use it in notebook, I'm seeing this:

![image](https://user-images.githubusercontent.com/36517077/76780300-ce1e8880-677a-11ea-8fd9-c83b7e9ba4b6.png)

![image](https://user-images.githubusercontent.com/36517077/76780385-f60dec00-677a-11ea-910d-6ddfcf716207.png)

![image](https://user-images.githubusercontent.com/36517077/76780441-0aea7f80-677b-11ea-9723-2ed635b4a59f.png)

Reply:
@Tikonderoga  why don't you go ahead and submit the new PR as it is and then we can also try it out locally ourselves to investivate

Reply:
@bryevdv 
https://github.com/bokeh/bokeh/pull/9789